src/main/java/com/timgroup/statsd/ServiceCheck.java

```diff
@@ -0,0 +1,149 @@
+package com.timgroup.statsd;
+
+/**
+ * A service check model, which is used to format a service check message
+ * sent to the datadog agent
+ */
+public class ServiceCheck {
+
+    private String name, hostname, message;
+
+    private int status, checkRunId, timestamp;
+
+    private String[] tags;
+
+    /**
+     */
+    public ServiceCheck() {
+    }
+
+    /**
+     * @param name
+     * @param status
+     */
+    public ServiceCheck(String name, int status) {
```

Comment:
Good idea. Probably just tags and message, because hostnames are automatically added in dd-agent.